lib/rack/contrib/profiler.rb

```diff
@@ -61,14 +64,32 @@ def profiling?(env)
         end
       end
 
+      # How many times to run the request within the profiler.
+      # If the profiler_runs query parameter is set, use that.
+      # Otherwise, use the :times option passed to `#initialize`.
+      # If the profiler_runs query parameter is greater than the
+      # :maximum option passed to `#initialize`, use the :maximum
+      # option.
+      def runs(request)
+        if profiler_runs = request.params['profiler_runs']
+          profiler_runs = profiler_runs.to_i
+          if profiler_runs > @maximum_runs
+            return @maximum_runs
+          else
+            return profiler_runs
+          end
```

Comment:
Any reason not to go with this?

```suggestion
          [profiler_runs.to_i, @maximum_runs].min
```

Could also drop the `return`s if you were feeling... implicit.

Reply:
Without branch coverage, the terse form is tricky to test in a way that gives confidence that every avenue is executed. That being said, we aren't testing coverage... but we should be :)